lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.CoolingTower.rb

```diff
@@ -1,110 +0,0 @@
-# A variety of cooling tower methods that are the same regardless of type.
-# These methods are available to CoolingTowerSingleSpeed, CoolingTowerTwoSpeed, and CoolingTowerVariableSpeed
-module ASHRAEPRMCoolingTower
-  # @!group CoolingTower
-
-  # Set the cooling tower fan power such that the tower
-  # hits the minimum performance (gpm/hp) specified by the standard.
-  # Note that in this case hp is motor nameplate hp, per 90.1.
-  # This method assumes that the fan brake horsepower is 90%
-  # of the motor nameplate hp.
-  # This method determines the minimum motor efficiency
-  # for the nameplate motor hp and sets the actual
-  # fan power by multiplying the brake horsepower
-  # by the efficiency.  Thus the fan power used as
-  # an input to the simulation divided by the design flow
-  # rate will not (and should not)
-  # exactly equal the minimum tower performance.
-  #
-  # @param cooling_tower [OpenStudio::Model::StraightComponent] cooling tower object, allowable types:
-  #   CoolingTowerSingleSpeed, CoolingTowerTwoSpeed, and CoolingTowerVariableSpeed
-  # @return [Bool] returns true if successful, false if not
-  def cooling_tower_apply_minimum_power_per_flow(cooling_tower)
-    # Get the design water flow rate
-    design_water_flow_m3_per_s = nil
-    if cooling_tower.designWaterFlowRate.is_initialized
-      design_water_flow_m3_per_s = cooling_tower.designWaterFlowRate.get
-    elsif cooling_tower.autosizedDesignWaterFlowRate.is_initialized
-      design_water_flow_m3_per_s = cooling_tower.autosizedDesignWaterFlowRate.get
-    else
-      OpenStudio.logFree(OpenStudio::Warn, 'openstudio.standards.CoolingTower', "For #{cooling_tower.name} design water flow rate is not available, cannot apply efficiency standard.")
-      return false
-    end
-    design_water_flow_gpm = OpenStudio.convert(design_water_flow_m3_per_s, 'm^3/s', 'gal/min').get
-
-    # Get the table of cooling tower efficiencies
-    heat_rejection = standards_data['heat_rejection']
-
-    # Define the criteria to find the cooling tower properties
-    # in the hvac standards data set.
-    search_criteria = {}
-    search_criteria['template'] = template
-
-    # By definition cooling towers in E+ are open.
-    # Closed cooling towers are the fluidcooler objects.
-    search_criteria['equipment_type'] = 'Open Cooling Tower'
-
-    # Define the criteria to find the fan type
-    search_criteria['fan_type'] = 'Axial'
```

Comment:
I generally agree this function is largely duplicate with standard.
To remove this function, we should also update the [data file](https://github.com/NREL/openstudio-standards/blob/AppendixG_Dev/lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm_2019/data/ashrae_90_1_prm_2019.heat_rejection.json#L6) to correctly extract the performance.

Once the data file is corrected, the hvac_efficiency test can pass the CI